packages/@aws-cdk/aws-lambda-nodejs/test/bundling.test.ts

```diff
@@ -181,6 +181,44 @@ test('esbuild bundling with esbuild options', () => {
   });
 });
 
+test('esbuild bundling with esbuild more options', () => {
```

Comment:
would update the previous test (`esbuild bundling with esbuild options`) which specifically tests all the `esbuild` options.

Reply:
I had a thought to do the same but the contributing doc says prefer not to modify the existing one!

Reply:
you are right, but I think that here it would be fine.